spotseeker_server/models.py

```diff
@@ -211,6 +210,12 @@ def get_with_external(spot_id):
         else:
             return Spot.objects.get(pk=spot_id)
 
+    def delete(self):
```

Comment:
I think we used the etag approach to support multiple ss_servers pointed to one DB, since a server can only invalidate its own cache.